estate/models/estate_property.py

```diff
@@ -0,0 +1,27 @@
+from odoo import fields, models
+
+
+class RecurringPlan(models.Model):
```

Comment:
```suggestion
class EstateProperty(models.Model):
```
The Python class name should follow the `CamelCase` version of the model's `_name`.